R/data.table.R

```diff
@@ -49,6 +49,32 @@ null.data.table = function() {
   setalloccol(ans)
 }
 
+rowwiseDT = function(..., key=NULL) {
+  x = substitute(list(...))[-1L]
+  if (is.null(nms <- names(x)))
+    stop("Must provide at least one column (use `name=`)")
```

Comment:
Add ```See ?rowwiseDT``` ?? If a user does not know to do ```name = ``` that likely means they have no idea about the function. 

Also, to channel Michael, I think it is common to use```stop(gettextf("stop message", domain = "R-data.table"))``` for translation purposes.

Reply:
Actually in this case, no need -- `tools::update_pkg_po` includes all literal strings within `stop()`, so this would be covered already. `gettextf` is for when there is some `sprintf` formatting to do, because something like

```
stop("looking for ",n," elements but only received ",m)
```

gets turned into two translations:

```
looking for 
elements but only received
```

which in general makes the translation process more difficult

Reply:
That is helpful. What does ```domain = ``` do and does the stop message with ```sprintf``` a couple of lines down need ```gettextf(...)```

Reply:
yes, good catch & fixed on `stop(sprintf())`.

`domain` supplies the name of a `.pot` file where to find the message ID, in turn used to look up the translation. It's _supposed_ to "just work" & we don't need to specify it:

> If `domain` is `NULL` or `""`, and `gettext` or `ngettext` is called from a function in the namespace of package `pkg` the domain is set to `"R-pkg"`. Otherwise there is no default domain.

However that wasn't working when I was trying to get translations going (I could be wrong or I could be doing something wrong)